sycl/include/CL/sycl/detail/builtins.hpp

```diff
@@ -19,9 +20,23 @@
 #ifdef __SYCL_DEVICE_ONLY__
 #define __FUNC_PREFIX_OCL  __spirv_ocl_
 #define __FUNC_PREFIX_CORE  __spirv_
+#define EXTERN_IT1(Ret, prefix, call, Arg1)
+#define EXTERN_IT2(Ret, prefix, call, Arg1, Arg2)
+#define EXTERN_IT3(Ret, prefix, call, Arg1, Arg2, Arg3)
+#define TPARAMS1(Ret, A1) <Ret, A1>
+#define TPARAMS2(Ret, A1, A2) <Ret, A1, A2>
+#define TPARAMS3(Ret, A1, A2, A3) <Ret, A1, A2, A3>
```

Comment:
Please, use `__SYCL_` prefix for defines in SYCL header files to not break following valid user code:
```c++
#define EXTERN_IT1
#include <CL/sycl.hpp>
```